design/FY2021/NFP-runarouind-heat-recovery-loop.md

```diff
@@ -0,0 +1,143 @@
+Run-around Heat Recovery Loop
+================
+
+**R. Raustad, FSEC**
+
+ - Original Date - 2/5/21
+ - Revision Date
+ 
+
+## Justification for New Feature ##
+
+Heat recovery is common in HVAC systems. A heat exchanger can easily be installed in an air system where two air steams are side-by-side. However, when the air steams are not are not located near each other, application of heat recovery is more difficult. Designers have overcome this issue by using water coils connected with piping and a pump to create a closed loop system where energy is transfered between these two coil. Third parties are not able to model this type of system in EnergyPlus.
+
+## E-mail and  Conference Call Conclusions ##
+
+
+
+## Overview ##
+
+The following is an except of the abstract from <http://what-when-how.com/energy-engineering/run-around-heat-recovery-systems-energy-engineering/>.
+
+Run-around heat-recovery systems are often used to recover heat from the exhaust air in building ventilation systems, particularly in cold climates. In a typical heat-recovery system, an ethylene glycol and water solution is used as a “coupling fluid” to prevent the system from freezing. The design of a run-around heat-recovery system involves consideration of the heat-transfer rates between the fluids. A challenging problem is how to significantly increase those rates between the fluids, while using less energy for heating and cooling loads in buildings. Due to the heat-transfer characteristics of this coupling fluid and the operating and capital cost factors, the typical overall effectiveness of such systems is only about 50%. Studies show that two-phase, gas-liquid coupling fluids have much higher convective heat-transfer rates than single-phase flows at the same mass flow rates.
+
+The following figures and text were provided in the Trane TRACE 700 User’s Manual • CDS-PRM001-EN at <https://software.trane.com/RightNow/0103-CoilLoopEnergyRecovery/Coil_loop_exhaust-air_energy_recovery.pdf>. 
+
+A simple schematic of a run-around coil heat recovery loop is shown in Figure 1. Insert A and B show heat recovery between two air stream while insert C shows energy transfer from before to after a cooling coil (similar to `CoilSystem:Cooling:DX:HeatExchangerAssisted`. 
+
+![RunaroundCoilLoop](RunaroundCoilHXLoop.png)
+## *Figure 1. Run around coil loop schematic.* ##
+
+Figure 2 shows an example run-around coil design for winter and summer operation. During the heating season (Inset A), heat extracted from the exhaust air stream (EA)
```

Comment:
Exhaust air stream is interesting.  If there is scope would be interesting to be able to put a coil in the outlet of each zone exhaust fan.

Reply:
I have a related new feature task for "Dedicated Airstreams to Model General Exhaust Fans" so it should be possible to design that to work with this.